setup.cfg

```diff
@@ -91,7 +91,9 @@ tests =
     pytest-cov
     pre-commit
 mypy =
-    mypy>=0.950
+    # mypy-1.0 crashing https://github.com/python/mypy/issues/14631
+    # TODO: unpin the version once above issue resolved
+    mypy>=0.950, <1.0.0
```

Comment:
Can we create a follow up ticket to track the bug fix and unpin this upper bound?

Reply:
https://github.com/astronomer/astronomer-providers/issues/868